htdocs/comm/propal.php

```diff
@@ -2252,10 +2252,16 @@
 		// Linked object block
 		$somethingshown = $form->showLinkedObjectBlock($object);
 
-		// Show links to link elements
-		$linktoelem = $form->showLinkToObjectBlock($object);
-		if ($linktoelem) print '<br>'.$linktoelem;
```

Comment:
The goal of this function is to show ALL linked objects, whatever is the type.
So why intoducing specific code for intervention ? If intervention is linked to proposal, it should appears with this generic code with no need of change (if not, pb is inside function showLinkToObjectBlock)